hazelcast/src/test/java/com/hazelcast/jet/core/JetTestSupport.java

```diff
@@ -440,28 +444,28 @@ private static void ditchJob0(@Nonnull Job job, @Nonnull HazelcastInstance... in
                     job.join();
                 } catch (JobNotFoundException e) {
                     SUPPORT_LOGGER.fine("Job " + job.getIdString() + " is gone.");
-                    return;
+                    return false;
```

Comment:
We managed to execute ```cancel()```, but we fail on ```join()```. Shouldn't we treat it as successful ditch and return ```true```? cc: @frant-hartm

Reply:
After ditch0() call, the test suite is trying to assert that the job is not runnning. Therefore any JobNotFoundException in ditch0() is a failed ditch.

Reply:
I've added a try/catch block to the  assertTrueEventually() that checks if the job is running or not. There is no need to check the return result of ditchJob0().